cpu/rpx0xx/include/cpu_conf.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup         cpu_rpx0xx
+ * @{
+ *
+ * @file
+ * @brief           Implementation specific CPU configuration options
+ *
+ * @author          Fabian Hüßler <fabian.huessler@ovgu.de>
+*/
+
+#ifndef CPU_CONF_H
+#define CPU_CONF_H
+
+#include <stdint.h>
+
+#include "macros/units.h"
+#include "cpu_conf_common.h"
+#include "vendor/RP2040.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   ARM Cortex-M specific CPU configuration
+ * @{
+ */
+#define CPU_DEFAULT_IRQ_PRIO            (1U)
+#define CPU_FLASH_BASE                  ROM_START_ADDR
```

Comment:
ROM_START_ADDR is only defined as a `make` variable, not a C define.

Reply:
Did so via `CFLAGS += -D` to avoid having to sync to numbers, in case something changes.